### PR TITLE
JS Practices

### PR DESCRIPTION
**Pull request by itsmraga-hub**
# 💯 Achievements.

- No linter errors.
- JavaScript Best Practices.

### CODE REVIEW

File: src/modules/addRemove.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  constructor(description, index = 0, isComplete = false) {
+    this.description = description;
+    this.isComplete = isComplete;
+    this.index = index;
+  }
+
+  addTask = (task, tasks) => {
+    tasks.push(task);
+    localStorage.setItem('tasks', JSON.stringify(tasks));
+  }
```
Comment (samiullah997) [inline src/modules/addRemove.js]: image
make change your empty submitting task.

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+  width: 100%;
+}
+
+.main-container .container .btn {
+  width: 100%;
+  background-color: #f1f2f5;
+  font-size: 1.2em;
+  border: none;
+  padding: 0.5em;
+}
```
Comment (samiullah997) [inline src/style.css:145]: image
when the checkbox is checked then make your button text-decoration underline.

File: src/modules/addRemove.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+}
+
+const loadFromStorage = (tasks) => {
+  tasks = JSON.parse(localStorage.getItem('tasks')) || [];
+  for (let i = 0; i < tasks.length; i += 1) {
+    tasks[i].index = i + 1;
+  }
+  return tasks;
+};
```
Comment (samiullah997) [inline src/modules/addRemove.js]: image
please start your task index from 1.

**Review by samiullah997 — CHANGES_REQUESTED** [review]
Hi @itsmraga-hub  👋🏿 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost


## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/style.css (hunk 2 above)
Comment (itsmraga-hub) [reply src/style.css:145]: Thanks for noticing this. Though I think this is just a styling preference.

File: src/modules/addRemove.js (hunk 3 above)
Comment (itsmraga-hub) [reply src/modules/addRemove.js]: Thanks for noticing this bug. This was something that I had definitely missed. Consider it worked on.

File: src/modules/addRemove.js (hunk 1 above)
Comment (itsmraga-hub) [reply src/modules/addRemove.js]: This was also something I did not notice. Consider it worked on.

**Review by samiullah997 — APPROVED** [review]
Hi @itsmraga-hub  ,👋🏻👋🏻
Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉🔥🔥💯💯

Yx9v



Highlights ✨✨
📌 Descriptive pull request✅
📌 Great working on the setup ✅

Optional suggestions
Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**